src/Mailgun/Resource/Api/SimpleResponse.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * Copyright (C) 2013-2016 Mailgun.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+namespace Mailgun\Resource\Api;
+
+/**
+ * @author Sean Johnson <sean@mailgun.com>
+ */
+class SimpleResponse
```

Comment:
No, this is most literally the simplest response the Mailgun API will return. There are quite a few calls that return a simple response, such as {"message": "..."} or a rate-limit response {"retry-seconds": ...}. We could attempt to add an `httpResponseCode` member, but we would either need to have a common abstraction for all of the classes that are a "response" that also grabs / handles the `httpResponseCode` or handle it in some other way.